.gitlab-ci.yml

```diff
@@ -123,8 +123,9 @@ clippy-std:
   needs:
     - check-std
   script:
+    - rustup default nightly-2020-02-01
```

Comment:
Please remove these lines. Not needed and just confusing.

Reply:
Default toolchain in the docker image is `nightly-2020-02-17`.
If remove these overrides, it will use clippy from that toolchain.
Moreover, this workaround is not forever here.

Reply:
No it won't since we specified the `nightly-2020-02-01` toolchain below with `+nightly-2020-02-01` at the `clippy` invokation.

Reply:
ah of course, thanks